spec/services/doc_auth/acuant/acuant_client_spec.rb

```diff
@@ -144,6 +144,33 @@
 
         expect(result.success?).to eq(true)
       end
+      it 'notes that address line 2 was present' do
+        instance_id = 'this-is-a-test-instance-id'
+        url = URI.join(assure_id_url, "/AssureIDService/Document/#{instance_id}")
+        stub_request(:get, url).to_return(body: AcuantFixtures.get_results_response_success)
+
+        result = subject.get_results(instance_id: instance_id)
+
+        expect(result.extra).to include(address_line2_present: true)
+      end
+
+      context 'when there is no address line 2' do
+        it 'notes that address line 2 was not present' do
+          instance_id = 'this-is-a-test-instance-id'
+          url = URI.join(assure_id_url, "/AssureIDService/Document/#{instance_id}")
+          stub_request(:get, url).to_return(
+            body: begin
+              json = JSON.parse(AcuantFixtures.get_results_response_success)
+              json['Fields'] = json['Fields'].select { |f| f['Name'] != 'Address Line 2' }
+              JSON.generate(json)
+            end,
+          )
```

Comment:
looks like a static response so prob no need for the `begin...end`? and could even throw a `.tap` in there if ya fancy

```suggestion
          body = JSON.parse(AcuantFixtures.get_results_response_success).tap do |json|
              json['Fields'] = json['Fields'].select { |f| f['Name'] != 'Address Line 2' }
           end.to_json
          stub_request(:get, url).to_return(body: body)
```